integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -499,6 +499,9 @@ object CarbonDataRDDFactory {
       LoadTablePostExecutionEvent(sqlContext.sparkSession,
         carbonTable.getCarbonTableIdentifier,
         carbonLoadModel)
+      // REVIEW_COMMENT: I think here we should know the datamap loading is failed or sucess,
```

Comment:
Kunal is working on it ...if any table loading fails it will fail complete data loading process